extra/1-radio-stations.js

```diff
@@ -26,6 +34,12 @@
  */
 // `getStations` goes here
 
+function getStations() {
+  let arrLength = getAllFrequencies().length;
+
+  return getAllFrequencies().filter((item) => isRadioStation(item));
```

Comment:
perfect. You can drop the `()` when an inline function has only one parameter:
`item => isRadioStation(item)`

Reply:
In fact, and feel free to ignore this comment, you can write:
`return getAllFrequencies().filter(isRadioStation));`
Because 
```
(item) => isRadioStation(item)
```
is the same as
```
isRadioStation
```

Reply:
Again, don't worry if it seems obscure.